README.rst

```diff
@@ -52,19 +54,22 @@ To something like this:
         return Ok(user)
 
     user_result = get_user_by_email(email)
-    if user_result.is_ok():
+    if isinstance(user_result, Ok):
+        # type(user_result.value) == User
         do_something(user_result.value)
     else:
-        raise RuntimeError('Could not fetch user: %s' user_result.value)
+        # type(user_result.value) == str
+        raise RuntimeError('Could not fetch user: %s' % user_result.value)
 
 As this is Python and not Rust, you will lose some of the advantages that it
 brings, like elegant combinations with the ``match`` statement. On the other
 side, you don't have to return semantically unclear tuples anymore.
 
 Not all methods (https://doc.rust-lang.org/std/result/enum.Result.html) have
-been implemented, only the ones that make sense in the Python context. You still
-don't get any type safety at runtime, but some easier handling of types that can
-be OK or not, without resorting to custom exceptions.
+been implemented, only the ones that make sense in the Python context. By using
+``isinstance`` to check for ``Ok`` or ``Err`` you get type safe access to the
+stored values. All of this in a package allowing easier handling of values
+that can be OK or not, without resorting to custom exceptions.
```

Comment:
You only get type safety when using mypy. Maybe that should be mentioned.

Reply:
Fixed in 688711e with your suggestion!